### PR TITLE
Implement all /role subcommands

### LINKED ISSUE #17: Setup moderation commands and interface with the db
Needed:
- [x] /setModeratorRole
- [x] /setAlliesRole
- [x] /setAxisRole
- [x] /setSpectatorRole
- [ ] /verifyAll
- [x] /setLogChannel
- [x] /setVerifyChannel
- [x] /setTroopMovementChannel
- [x] /setFacilityChannel
- [x] /addCommandChannel

### PR DESCRIPTION
**Pull request by TotallyNot**
partially resolves #17

### CODE REVIEW

File: src/handler/guild/role.ts [hunk 1]
```diff
@@ -138,128 +130,184 @@ export class Role {
...
+    async adminRoleAdd(
+        interaction: CommandInteraction,
+        roleID: string
+    ): Promise<void> {
+        await getConnection()
+            .createEntityManager()
+            .transaction(async (manager) => {
+                const guildEntity = await manager.findOneOrFail(
+                    GuildEntity,
+                    interaction.guildId
```
Comment (Natty-Boh) [inline src/handler/guild/role.ts:151]: why are you not use the repository pattern here?

Comment (TotallyNot) [reply src/handler/guild/role.ts:151]: best as I can tell that doesn't work with database transactions

**Review by Natty-Boh — APPROVED** [review]
looks fine, although i do think you should implement a custom guild repository. imo should either always follow the pattern or never. half and half is awkward. but im fine either way i guess

File: src/handler/guild/role.ts (hunk 1 above)
Comment (Natty-Boh) [reply src/handler/guild/role.ts:151]: https://github.com/typeorm/typeorm/issues/1521 seems like it maybe kind of does but not really in a very useful way so i guess maybe skip it lol

**Pull request merged**